src/test/java/kg/jarkyn/cobspecserver/doubles/ClientDouble.java

```diff
@@ -0,0 +1,29 @@
+package kg.jarkyn.cobspecserver.doubles;
+
+import kg.jarkyn.cobspecserver.Client;
+
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class ClientDouble implements Client {
```

Comment:
For convenience I'd probably add a `getString()` or `getContent()` method that returns me the string content directly, instead of having to call that in my tests.